### PR TITLE
[GCP] Add storage data stream

### PR DESCRIPTION
**Pull request by gpop63**
## What does this PR do?

Adds `storage` data stream.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Closes: #3717

### CODE REVIEW

File: packages/gcp/changelog.yml [hunk 1]
```diff
@@ -32,6 +32,9 @@
     - description: Add GCP Firestore Data stream
       type: enhancement
       link: https://github.com/elastic/integrations/pull/2704
+    - description: Add GCP Storage Data Stream
```
Comment (endorama) [inline packages/gcp/changelog.yml:35]: Please target the most recent version of this package

**Review by endorama — CHANGES_REQUESTED** [review]
Small change but LGTM!

File: packages/gcp/changelog.yml (hunk 1 above)
Comment (gpop63) [reply packages/gcp/changelog.yml]: Fixed.

**Review by endorama — APPROVED** [review]

**Comment by endorama** [comment]
I cannot reproduce the failure locally, as the same check (`elastic-package check -v`) does not produce any error.

**Review by tommyers-elastic — CHANGES_REQUESTED** [review]
small typo in metric name

**Review by tommyers-elastic — APPROVED** [review]
LGTM

**Pull request merged**